Ports/tar/patches/0001-Disable-Wunused-parameter-warnings.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Liav A <liavalb@gmail.com>
+Date: Fri, 11 Aug 2023 14:59:46 +0300
+Subject: [PATCH] Disable -Wunused-parameter warnings
+
+---
+ m4/gnulib-common.m4 | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/m4/gnulib-common.m4 b/m4/gnulib-common.m4
+index 2426eb639852f5e7d07e1c04e609cb174f2e3498..4a528b8dfa0c5b5f76d3125fbe7e21215fa5e7be 100644
+--- a/m4/gnulib-common.m4
++++ b/m4/gnulib-common.m4
+@@ -1070,7 +1070,7 @@ AC_DEFUN([gl_CC_GNULIB_WARNINGS],
+       -Wno-sign-compare
+       -Wno-undef
+       -Wno-unused-function
+-      -Wno-unused-parameter
```

Comment:
I'm a bit confused. Doesn't this line already disable the `-Wunused-parameter` warning already?

Reply:
Should be OK now :)

Reply:
This is confusing for two reasons actually. First obviously being that we are actually removing the option that is supposed to suppress those errors here. The second is that we are changing an input file for `autoconf` here, so nothing that we do here would affect the build without running `autoconf` first.

Reply:
I now ensured that in addition to just removing the `-Wno-unused-parameter` flag, we do it in the `configure` file :)